src/netspresso_trainer/models/op/custom.py

```diff
@@ -171,6 +171,69 @@ def forward(self, x: Union[Tensor, Proxy]) -> Union[Tensor, Proxy]:
         x = self.final_act(x)
         return x
 
+class RepConv(nn.Module):
+    """
+    A convolutional block that combines two convolution layers (kernel and point-wise conv).
+    """
```

Comment:
I think renaming `RepVggBlock` -> `RepConv` is not proper to this class.
Actually, RepVGG block is came from https://arxiv.org/abs/2101.03697, this block is named by author.
Also, `RepVggBlock` should have residual connection option according to the original paper.

And, please add original codebase reference so that we can keep an eye on code license issue.

Reply:
Thank you for the suggestions! I've include your feedback via additional commits👍